unified-prototype/unified-plugin/build-update-utils/build.gradle.kts

```diff
@@ -0,0 +1,56 @@
+description = "Utility tasks for CI to use to release a version of the plugin"
+
+val ciUpdateTasks = "CI Update Tasks"
+
+val versionFile = rootProject.file("version.txt")
+
+open class ExecOperationsHelper @Inject constructor(val execOperations: ExecOperations) {
```

Comment:
Can we just move this all into a proper plugin and apply it to the root of unified-plugin? That way this doesn't need to use workarounds to get ExecOperations and we can put the tasks in the proper project (vs a subproject).

Reply:
Done.